R/fso_px_url.R

```diff
@@ -0,0 +1,13 @@
+#' Get pxt download url for px id
+#'
+#' @param px_id identifier for px cube
+#'
+#' @return download_url
+#' @export
+#'
+#' @examples get_fso_px_download_url('px-x-0102020207_102')
+get_fso_px_download_url <- function(px_id) {
+  fso_px_url <- 'https://www.pxweb.bfs.admin.ch/DownloadFile.aspx?file='
+  download_url <- paste0(fso_px_url, px_id)
```

Comment:
```suggestion
  download_url <- paste0(fso_px_url, px_id)
  check_px_cube_url(download_url)
```

Once created, we need to check that the initial object given was a valid px_id. OR we need an extra small function that checks the format of the px_id. This could be better, that way any px_id coming in from anywhere gets checked. It would also allow to throw a clear error message to the user saying "your px_id needs to be in this format: XXXX"